doc/html/multilayer.html

```diff
@@ -18,7 +18,7 @@ <H1 ALIGN=Center>
   command is placed in a separate layer and may be edited in isolation.
   <P>
   A new font may be converted to a multi-layered font by selecting
-  Element-&gt;Font Info-&gt;Layers-&gt;Multi Layered Font.
+  Element-&gt;Font Info-&gt;Layers-&gt;All layers cubic and Type3 Multi Layered Font.
```

Comment:
What I meant by this, is that there are two options to select in order to convert the font to multi layers.  If "All layers cubic" is not selected, the "Type3 Multi Layer" option is not available.

Feel free to rewrite this, if it is unclear.
![mutli-layer-options](https://cloud.githubusercontent.com/assets/627071/19937526/0c794612-a0f8-11e6-983b-99b88d516a16.png)